src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
+Animal.remove = function() {
+  Animal.alive = Animal.alive.filter((obj) => obj.health !== 0);
```

Comment:
`obj ` is not the best naming
```suggestion
  Animal.alive = Animal.alive.filter((animal) => obj.health !== 0);
```